src/Core/src/Handlers/RefreshView/RefreshViewHandler.Android.cs

```diff
@@ -28,6 +28,30 @@ protected override void DisconnectHandler(MauiSwipeRefreshLayout platformView)
 			base.DisconnectHandler(platformView);
 		}
 
+		public override Size GetDesiredSize(double widthConstraint, double heightConstraint)
+		{
+			var Context = MauiContext?.Context;
+			var platformView = PlatformView;
+			var virtualView = VirtualView;
+
+			if (double.IsInfinity(virtualView.MaximumWidth) || double.IsInfinity(virtualView.MaximumHeight))
+			{
+				if (platformView == null || virtualView == null || Context == null)
```

Comment:
I'm not sure the cases each of these can actually be null in calls to this method, but you are using virtualView before null checking it here. Either the check for virtualView is pointless or all the null checks should probably be moved up / before the main if. And maybe just skip the main logic and call base instead of returning zero?